mypyc/ir/rtypes.py

```diff
@@ -234,9 +234,14 @@ def __repr__(self) -> str:
 short_int_rprimitive = RPrimitive('short_int', is_unboxed=True, is_refcounted=False,
                                   ctype='CPyTagged')  # type: Final
 
-# low level integer (corresponds to C's 'int').
-c_int_rprimitive = RPrimitive('c_int', is_unboxed=True, is_refcounted=False,
+# low level integer (corresponds to C's 'int's).
+c_int32_rprimitive = RPrimitive('c_int32', is_unboxed=True, is_refcounted=False,
+                              ctype='int')  # type: Final
+c_int64_rprimitive = RPrimitive('c_int64', is_unboxed=True, is_refcounted=False,
                               ctype='Py_ssize_t')  # type: Final
+# integer alias
+c_int_rprimitive = c_int32_rprimitive
+c_pyssize_t_rprimitive = c_int64_rprimitive
```

Comment:
The definition should depend on the platform. If we are on a 64-bit platform, it's expected to be `int64_rprimitive`, but on a 32-bit platform it's `int32_rprimitive`. (We don't care about exotic platforms such as 32-bit addresses with 64-bit registers.)

Reply:
Do we have ways of detecting the current platform?

Reply:
You can use something like `sys.maxsize < (1 << 31)` to check if we are on a 32-bit platform.